src/SFML/Graphics/Texture.cpp

```diff
@@ -120,6 +121,49 @@ Texture::~Texture()
     }
 }
 
+////////////////////////////////////////////////////////////
+Texture::Texture(Texture&& right) noexcept :
+m_size(std::exchange(right.m_size, {0U, 0U})),
+m_actualSize(std::exchange(right.m_actualSize, {0U, 0U})),
```

Comment:
```suggestion
m_size(std::exchange(right.m_size, {})),
m_actualSize(std::exchange(right.m_actualSize, {})),
```
No need to repeat the default values of a vector.